com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/runtime.py

```diff
@@ -18,6 +18,9 @@ def setupOperator(dir):
 
 def pickleObject(v):
     return pickle.dumps(v)
+
```

Comment:
Not required to add such a function, it's wrapping one static function in another static function for no benefit.